shopify/resources/inventory_level.py

```diff
@@ -0,0 +1,40 @@
+from ..base import ShopifyResource
+import shopify
+import json
+
+
+class InventoryLevel(ShopifyResource):
+
+    def __repr__(self):
+        return '%s(%s)' % (self._singular, self.available)
+
+    @classmethod
+    def adjust(cls, location_id, inventory_item_id, available_adjustment):
+        body = {
+            'inventory_item_id': inventory_item_id,
+            'location_id': location_id,
+            'available_adjustment': available_adjustment
+        }
+        resource = cls.post('adjust', body=json.dumps(body))
```

Comment:
The `body` should be encoded, this throws an error:
```
TypeError: POST data should be bytes, an iterable of bytes, or a file object. It cannot be of type str.
```

Changing this line to:
```python
resource = cls.post('adjust', body=json.dumps(body).encode())
```
fixes the error for me.

Reply:
Good catch. I'll fix this in #245